docs/editors/debugging.md

```diff
@@ -0,0 +1,29 @@
+---
+id: debugging
+sidebar_label: Debugging
+title: Debugging using Debug Adapter Protocol
+---
+
+Metals implements the Debug Adapter Protocol, which can be used by the editor to
+communicate with JVM to run and debug code.
+
+## How to add support for debugging in my editor?
```

Comment:
Though it's not only for new editors - all the existing ones do not implement it. I think it is useful to have it separate, since might be easier to find for people trying to implement it for Vim/Emacs etc.

Reply:
`new-editor.md` is the reference document even for editors like vim/emacs that we already support but don't implement all metals features.